src/libs/TransactionUtils.ts

```diff
@@ -356,8 +356,8 @@ function getMerchant(transaction: OnyxEntry<Transaction>): string {
     return transaction?.modifiedMerchant ? transaction.modifiedMerchant : transaction?.merchant ?? '';
 }
 
-function getDistance(transaction: Transaction): number {
-    return transaction?.routes?.route0?.distance ?? 0;
+function getDistance(transaction: Transaction | null): number {
+    return transaction?.comment?.customUnit?.quantity ?? 0;
```

Comment:
Coming from https://github.com/Expensify/App/issues/41389. With this change we also needed to tweak this

https://github.com/Expensify/App/blob/5f504301f9be63b728f78fdf2afd085a2872fbdd/src/components/MoneyRequestConfirmationList.tsx#L266

in order to adjust the quantity whenever there's a modification in the distance. More details [here](https://github.com/Expensify/App/issues/41389#issuecomment-2088186951).